packages/amazon-bedrock/src/bedrock-chat-language-model.ts

```diff
@@ -111,24 +116,10 @@ export class BedrockChatLanguageModel implements LanguageModelV2 {
 
     const { system, messages } = convertToBedrockChatMessages(prompt);
 
-    // Parse thinking options from provider metadata
-    const reasoningConfigOptions =
-      BedrockReasoningConfigOptionsSchema.safeParse(
-        providerOptions?.bedrock?.reasoning_config,
```

Comment:
here we weren't using camelCase even though docs said it

Reply:
let's fix that in `v5` and move to a pattern similar to groq where we parse all options first

Reply:
cool, done that in this PR lmk what you think about them. I'm going to pass them into AI to get it to do the rest of them hopefully